packages/babel-core/test/config-loading.js

```diff
@@ -166,6 +168,36 @@ describe("@babel/core config loading", () => {
     });
   });
 
+  describe("loadPartialConfigAsync", () => {
+    // https://github.com/babel/babel/issues/15916
+    (supportsESM ? it : it.skip)(
```

Comment:
Nit: may also use `itESM` from `$repo-utils`.

Reply:
itESM is for when we are compiling to ESM, while in this case I want the test to run also when compiling to CJS if ESM config files are supported.

Reply:
You are right, I didn't figure out it was actually `itGte("12")`.